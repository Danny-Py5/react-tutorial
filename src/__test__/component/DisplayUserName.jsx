export default function DisplayUserName({ userName }) {
  return (
    <div>
      <span>{userName}</span>
    </div>
  );
}
