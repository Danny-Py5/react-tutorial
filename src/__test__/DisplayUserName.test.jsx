import { it, describe, expect, beforeEach } from "vitest";
import { render, screen } from "@testing-library/react";
import DisplayUserName from "./component/DisplayUserName.jsx";

describe("DisplayUserName", () => {
  it("renders userName", () => {
    const result = render(<DisplayUserName userName={"Daniel"} />);
    expect(result.container).toMatchSnapshot();
  });
});
