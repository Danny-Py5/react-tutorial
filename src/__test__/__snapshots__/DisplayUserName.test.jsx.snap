// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DisplayUserName > renders userName 1`] = `
<div>
  <div>
    <span>
      Daniel
    </span>
  </div>
</div>
`;
